OrbitGl/DataManager.h

```diff
@@ -23,7 +23,8 @@ class DataManager final {
   void UpdateModuleInfos(uint32_t process_id,
                          const std::vector<ModuleInfo>& module_infos);
 
-  const std::vector<ModuleData*> GetModules(uint32_t process_id);
+  const ProcessInfo& GetProcessByPid(uint32_t process_id) const;
```

Comment:
Make it return ProcessData* - and add methods to ProcessData to access ProcessInfo field (convert it to wrapper), the way ModuleData/ModuleInfo now done.

It will also let us control modifiable/not-modifiable fields, for example name can only be read, where list of modules can be modified.

Reply:
Alright, please have a look.